app/lib/actions/foreman_ansible/play_host_roles.rb

```diff
@@ -0,0 +1,46 @@
+module Actions
+  module ForemanAnsible
+    class PlayHostRoles < Actions::EntryAction
+      include ::Actions::Helpers::WithContinuousOutput
+      include ::Actions::Helpers::WithDelegatedAction
+
+      def plan(host, proxy_selector = ::ForemanAnsible::ProxySelector.new)
```

Comment:
Update the foreman-tasks code from PR: I've updated them altogether  https://github.com/theforeman/foreman-tasks/pull/198/commits/a6cf005703e59d6d08edfd3343c1b91a5e000d2f